docs/source/zero-to-binderhub/setup-binderhub.rst

```diff
@@ -60,6 +60,11 @@ need to insert. Note that the first line is not indented at all::
         "client_x509_cert_url": "<REPLACE>"
       }
 
+If you are using Artifact Registry
```

Comment:
Can you change this to `Google Artifact Registry` (or `Google Cloud ...`, whatever's most appropriate), and similarly below? This will make the doc easier to read for people who aren't familiar with Google Cloud. Otherwise this looks good.

Reply:
Sure thing.